frontend/src/pages/modelRegistrySettings/ModelRegistriesTableRow.tsx

```diff
@@ -35,12 +45,18 @@ const ModelRegistriesTableRow: React.FC<ModelRegistriesTableRowProps> = ({
           <ModelRegistryTableRowStatus conditions={mr.status?.conditions} />
         </Td>
         <Td modifier="fitContent">
-          <Link
-            aria-label={`Manage permissions for model registry ${mr.metadata.name}`}
-            to={`/modelRegistrySettings/permissions/${mr.metadata.name}`}
-          >
-            Manage permissions
-          </Link>
+          {filteredRoleBindings.length === 0 ? (
+            <Tooltip content="You can manage permissions when the model registry becomes available.">
+              <span className="pf-v5-u-disabled-color-100">Manage permissions</span>
```

Comment:
Instead of the span here, we can use a PF Button with isAriaDisabled so it can receive keyboard focus.
```tsx
<Button isAriaDisabled variant="link" isInline>Manage permissions</Button>
```

For consistency, we should probably also use a `<Button variant="link" isInline>` in the enabled case below as well, just in case there are spacing differences. We can add an `onClick={() => navigate(...)}` instead of the `to` prop, and get `navigate` from `useNavigate()` imported from `react-router-dom`.

I'm actually not sure if we need `isInline` on them as well, whichever has the right spacing is fine.